src/content/docs/alerts-applied-intelligence/applied-intelligence/alert-coverage-gaps.mdx

```diff
@@ -0,0 +1,72 @@
+
+What does covering an entity mean? A covered entity means that you've set up an [alert](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/) to notify you of how a particular entity is performing. For example, you might have created an alert condition that opens an incident if a host's CPU goes above 80%. An uncovered entity is part of your system that is unmonitored which means there could be unhealthy behavior that goes unchecked. 
+
+We created [alert coverage gaps](https://one.newrelic.com/nrai/detection-gaps/home) to highlight those areas of your stack where you might be missing valuable data so you can prevent incidents before they happen. 
+
+## Find your coverage gaps [#find-your-coverage-gaps]
+
+On the Alerts and AI capabilites page, select alert coverage gaps in the naviagtion. There you will see a dashboard of all your entitites that are currently uncovered. 
+
+<img
+  title="Alert coverage gap dashboard."
+  alt="A screenshot of the alert coverage gap dashboard"
+  src={alertcoveragegaps}
+/>
+
+<figcaption>
+  When you select alert coverage gaps you will see a list of all your unmonitored entities.
+</figcaption>
+
+## Create an alert [#create-an-alert]
+
+Once you've selected an entity you'd like to monitor, then you can create an alert condition to notify your team of any abnormal behavior. 
+
+New Relic will dynamically suggest the conditions we believe should be monitored based on the selected entity and your specific system. Some [recommend alerts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/condition-recommendations/) we might be suggest are error percentage, apdex or response time. 
+
+Once you select an alert condition you'd like to monitor, then your team will be alerted when this signal deviates from its normal baseline, upper and lower, for at least 5 minutes by 3.00 standard deviation(s).
+
+<img
+  title="Create an alert."
+  alt="A screenshot of creating an alert from the alert coverage gap dashboard"
+  src={addanalert}
+/>
+
+<figcaption>
+  Creating an alert for an unmonitored entity.
+</figcaption>
+
+Every alert condition needs to be routed through a [policy](/docs/alerts-applied-intelligence/new-relic-alerts/learn-alerts/rules-limits-alerts) so you will be give the option to either select an existing policy or create a new one. 
+
+<img
+  title="Create a policy for alert coverage gaps."
+  alt="A screenshot of the alert coverage gap policy selection"
+  src={createapolicyalertcoveragegaps}
+/>
+
+<figcaption>
+  When you create an alert you will select which policy the alert will be grouped with.
```

Comment:
related to above: 'which policy the alert will be grouped with': I wonder if that's explaining the structural connection between alert condition and policy well. we might want to be consistent with how we explain that connection (e.g., a policy contains conditions, or whatever)